plugins/analysis/dashboard/frontend/src/app/components/Mana/Mana.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react'
+import {inject, observer} from "mobx-react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import {Badge, Card, Col} from "react-bootstrap";
+import ManaStore from "../../stores/ManaStore";
+import ManaGauge from "./ManaGauge";
+import ManaEventList from "./ManaEventList";
+import ManaLeaderboard from "./ManaLeaderboard";
+import ManaHistogram from "./ManaHistogram";
+
+interface Props {
+    manaStore: ManaStore;
+}
+
+@inject("manaStore")
+@observer
+export default class Mana extends React.Component<Props, any> {
+    render() {
+        let manaStore = this.props.manaStore
+        const {searchNode, searchTxID} = manaStore
+        return (
+            <Container>
+                <Row className={"mb-3 mt-3"}>
+                    <Col>
+                        <ManaGauge
```

Comment:
I don't think we should show this ManaGauge here, it is for a node, not for the whole network.

Reply:
Lets use this gauge to show the total mana in the network

Reply:
Good idea!